Hudl.Mjolnir.Tests/Command/Attribute/NewCommandAttributeAndProxyTestsIgnoringTimeouts.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Threading;
+using Hudl.Mjolnir.Command.Attribute;
+using Hudl.Mjolnir.Tests.Helper;
+using Xunit;
+
+namespace Hudl.Mjolnir.Tests.Command.Attribute
+{
+    public sealed class NewCommandAttributeAndProxyTestsIgnoringTimeouts : TestFixtureIgnoreTimeouts
```

Comment:
@robhruska Having started to move this logic to the start of each of the ~160 tests, I'm starting to think it's better where it is in the constructor. Here's my reason;

In each test, since we don't know what other tests may have done to the ConfigProvider, we have to initialise all of the possible keys. e.g. in a test where we're not looking to test anything related to the ignoreTimeouts functionality we still need to start the test off with a 
```csharp
ConfigProviderContext.Instance.SetConfigValue(ConfigProviderContext.IgnoreTimeoutsKey, false);
```
This seems like code bloat, and is perhaps more confusing than having it all in one place?
It also makes it tricky if we end up adding another config value to the tests!